packages/theme-tasks/lib/utils/nodesass-packageimporter.js

```diff
@@ -37,4 +31,30 @@ module.exports = function(url, prev/*, done*/) {
         file: file
     };
 
+}
+
+function slowPackageImporter(url) {
```

Comment:
May be refactored into a function that returns the needed importer to avoid duplication:

```js
const packageImporter = (options = { cache: false }) => url => {
    if (!url.startsWith("~")) {
        return null;
    }

    let file = path.resolve( path.join(
        process.cwd(),
        "node_modules/",
        url.slice(1)
    ) );

    if (options.cache && imported.has(file)) {
        return EMPTY_IMPORT;
    }

    imported.add(file);

    return { file };
}
```

Reply:
changes aaplied